src/renderer/sw_engine/tvgSwRasterTexmap.h

```diff
@@ -569,7 +569,7 @@ static void _rasterPolygonImageSegment(SwSurface* surface, const SwImage* image,
                     u += _dudx;
                     v += _dvdx;
                     //range over?
-                    if ((uint32_t)v >= image->h) break;
+                    if ((uint32_t)(int32_t)v >= image->h) break;
```

Comment:
this solve warning on mac. can we consider to do this in all places marked as "range over" ?
ex. Blending sample causes the same warning in l.382

Reply:
Yes, makes sense.